### PR TITLE
DDF-5317 Change serialized name in QueryBasic to "src"

### PR DESCRIPTION
**Pull request by jordanwilking**
#### What does this PR do?
Updates @SerializedName in QueryBasic, so hitting QueryMetacardApplication's endpoints will serialize "src" into QueryBasic's sources variable. 

#### Who is reviewing it? 
@Bdthomson 
@stevenmalmgren 
@vinamartin 

#### Ask 2 committers to review/merge the PR and tag them here.
@troymohl
@vinamartin

#### How should this be tested?

- Use Postman to POST to this endpoint: https://localhost:8993/search/catalog/internal/queries
  - You're sending "src", but the response should still come back as "sources". 
  - The body should look something like this:

```
{
    "cql": "(\"anyText\" ILIKE 'postman')",
    "filterTree": "{\"type\":\"AND\",\"filters\":[{\"type\":\"ILIKE\",\"property\":\"anyText\",\"value\":\"postman\"}]}",
    "excludeUnnecessaryAttributes": true,
    "count": 250,
    "start": 1,
    "federation": "selected",
    "sorts": [
        {
            "attribute": "modified",
            "direction": "descending"
        }
    ],
    "serverPageIndex": 1,
    "type": "advanced",
    "isLocal": false,
    "isOutdated": true,
    "spellcheck": false,
    "phonetics": false,
    "src": [
        "DDF"
    ],
    "id": "acba7b4c-1c9c-4a86-8ff7-eca0ea861be4",
    "totalHits": 0,
    "title": "postman"
}
```

#### What are the relevant tickets?
Fixes: #5317 

#### Screenshots
image

#### Checklist:
- [ ] Documentation Updated
  - N/A
- [ ] Update / Add Threat Dragon models
  - N/A
- [x] Update / Add Unit Tests
- [ ] Update / Add Integration Tests
  - N/A

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by jordanwilking** [comment]
build now

File: catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java [hunk 1]
```diff
@@ -76,7 +76,7 @@
   @SerializedName("enterprise")
   private Boolean enterprise;
 
-  @SerializedName("sources")
+  @SerializedName(value="sources", alternate="src")
```
Comment (Bdthomson) [inline catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java]: This is an okay temporary fix but it should just be one values. Can we just change the QueryBasic model to use `src` instead so it's consistent with the CQL endpoint?

Comment (jordanwilking) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java]: Can you confirm that no one else is using this endpoint in the other downstreams? I figured they're not, but I can't confirm it.

Comment (Bdthomson) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java]: @bellcc

Comment (jordanwilking) [reply catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/query/data/model/QueryBasic.java]: For posterity, @bellcc said it looks like it isn't being used in a way that will hurt anyone if it's changed to "src". Making the change.

**Comment by jordanwilking** [comment]
build now

**Comment by jordanwilking** [comment]
Missed a test that needed updated and Jenkins just gave me an "unidentified error" message. Build now.

**Comment by Bdthomson** [comment]
✅ Hero Successful
 
Verified:
- Body comes back with correct source key.

**Pull request merged**